### ISSUE TITLE
[bug]: Checkbox not clickable in CommandItem

### ISSUE THREAD

**Issue by V0IDL355** [shadcn-ui/ui#4475]
### Describe the bug

As soon as I add a checkbox to and CommandItem element its not clickable.
If u use tab to select the checkbox and press space it activates. U just can't click with a mouse

### Affected component/components

Checkbox, Command (CommandItem)

### How to reproduce

1. Put a checkbox inside CommandItem and try clicking it

### Codesandbox/StackBlitz link

https://codesandbox.io/p/devbox/kind-browser-hkkv8w?workspaceId=ded83c22-675d-472a-b064-f1637d1bca21

### System Info

```bash
Firefox
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by ItayBenAmi** [comment]
Hi, ran into this as well. It appears there is already an [open issue](https://github.com/shadcn-ui/ui/issues/2944) on the matter as well as a [pr](https://github.com/shadcn-ui/ui/pull/2945) with a suggested fix.

tl;dr: you can change `data-[disabled]` in CommandItem to `data-[disabled='true']`, which should work for your example.

**Comment by V0IDL355** [comment]
Thanks. Saw the pr https://github.com/shadcn-ui/ui/pull/2945 👍
